runtime/include/chpl-mem.h

```diff
@@ -265,29 +262,6 @@ void* chpl_mem_layerAlloc(size_t, int32_t lineno, int32_t filename);
 void* chpl_mem_layerRealloc(void*, size_t, int32_t lineno, int32_t filename);
 void chpl_mem_layerFree(void*, int32_t lineno, int32_t filename);
 
-//
```

Comment:
As noted above, I think that although chpl_mem_localizes() isn't needed once this change goes in, it will be useful in the near future and we should consider keeping it.  (If we do keep it now it reduces to compile-time constant, so it doesn't hurt execution time.)  chpl_mem_localizationThreshold() seems to be unused (oops!) so it can definitely go.